tests/testthat/test-get_rcode.R

```diff
@@ -18,7 +18,7 @@ testthat::test_that("get_rcode returns code from chunks at the end", {
   teal.code::chunks_push(id = "test", chunks = ch, quote(a <- 1))
 
   r_code_from_chunks <- strsplit(get_rcode(chunks = ch), "\n")[[1]]
-  r_code_from_header <- strsplit(sprintf("\n\n%s\n\na <- 1", paste(get_rcode_header(), collapse = "\n")), "\n")[[1]]
+  r_code_from_header <- strsplit(sprintf("\n\n%s\na <- 1", paste(get_rcode_header(), collapse = "\n")), "\n")[[1]]
```

Comment:
Lets not change this as it's being done here https://github.com/insightsengineering/teal/pull/800 ?